libs/blocks/commerce/commerce.js

```diff
@@ -0,0 +1,144 @@
+import { createTag, getConfig, loadScript } from '../../utils/utils.js';
+import { debounce } from '../section-metadata/section-metadata.js';
```

Comment:
See my other comment, this is not ideal imo.

Reply:
Fixed